pages/deployment/[uuid].tsx

```diff
@@ -187,30 +188,50 @@ export default function Deployment() {
   return (
     <>
       <Wrapper title={`Deployment: ${deployment.metadata.highLevelDetails.name}`} page='deployment'>
-        {hasUploadType && initialVersionRequested?.metadata.buildOptions.uploadType === ModelUploadType.Zip && (
-          <Box sx={{ textAlign: 'right', pb: 3 }}>
-            <Button variant='outlined' color='primary' startIcon={<Info />} onClick={handleClickOpen}>
-              Show download commands
-            </Button>
-          </Box>
-        )}
-        {hasUploadType && initialVersionRequested?.metadata.buildOptions.uploadType === ModelUploadType.ModelCard && (
-          <Box sx={{ pb: 2 }}>
-            <Alert
-              severity='info'
-              sx={{
-                width: 'fit-content',
-                m: 'auto',
-                backgroundColor: '#0288d1',
-                color: '#fff',
-                '& .MuiAlert-icon': {
-                  color: '#fff',
-                },
-              }}
+        {deployment && (
+          <Stack direction='row' alignItems='center' justifyContent='space-between'>
+            <Button
+              variant='text'
+              color='primary'
+              sx={{ pb: 3 }}
+              startIcon={<ArrowBackIosNewIcon />}
+              onClick={() => router.push(`/model/${(deployment.model as ModelDoc).uuid}`)}
```

Comment:
We should be type-guarding instead of using a type assertion (similar to casting in other languages) in situations like this.

That being said, I've created a new ticket (BAI-308) to address the root cause of this issue, so I'd argue this is fine for now so long as we address this in that ticket.